cmd/controller-manager/app/serve.go

```diff
@@ -31,17 +33,23 @@ import (
 	"k8s.io/component-base/configz"
 	"k8s.io/component-base/metrics/legacyregistry"
 	_ "k8s.io/component-base/metrics/prometheus/workqueue" // for workqueue metric registration
-	"k8s.io/kubernetes/pkg/api/legacyscheme"
 )
 
+func getLegacyschema() runtime.NegotiatedSerializer {
```

Comment:
I think we should have a different name. We're not getting a schema here we getting a codec/serializer. Also I think we're moving away from legacy here, which is a good thing. We also need to make sure we are using the same scheme here with all the GVKs.

Reply:
I am using scheme from client-go/kubernetes/scheme. Please have a look